lib/watir/attribute_helper.rb

```diff
@@ -38,6 +38,10 @@ def attribute_list
     #  @return [$1] value of $3 property
     #
     def attribute(type, method, attr)
+      if %i(thead tfoot).include? method
```

Comment:
If we remove it, then this code:
```
browser.table(id: 'table').thead
```
will be the same as this code: 
```
browser.table(id: 'table').attribute_value(:thead)
```
Instead of returning a `TableSection` instance as would be expected